shell/browser/ui/message_box_mac.mm

```diff
@@ -128,6 +133,12 @@ void ShowMessageBox(const MessageBoxSettings& settings,
     int ret = [[alert autorelease] runModal];
     std::move(callback).Run(ret, alert.suppressionButton.state == NSOnState);
   } else {
+    if (settings.id) {
+      if (base::Contains(g_dialogs, *settings.id))
```

Comment:
It may happen if someone calls `process._linkedBinding('electron_browser_dialog').showMessageBox` directly, or when `nextId` overflows while some old dialogs are still open.

I don't think either would ever happen, but it is an edge case anyway and a simple check can protect us from leaking things.